backend/lcfs/db/models/Issuance.py

```diff
@@ -0,0 +1,26 @@
+from sqlalchemy import Column, Integer, Sequence, BigInteger, ForeignKey, DateTime
+from sqlalchemy.orm import relationship
+
+from lcfs.db.base import BaseModel, Auditable, EffectiveDates
+
+class Issuance(BaseModel,Auditable, EffectiveDates):
+    __tablename__ = 'issuance'
+    __table_args__ = {'comment': "Goverment to organization compliance units issuance"}
+
+
+    id = Column(Integer, Sequence('issuance_id'), comment="Unique identifier for the issuance", primary_key=True, autoincrement=True)
+    compliance_units = Column(BigInteger, comment="Compliance Units")
+    organization_id = Column(Integer, ForeignKey('organization.id'))
+    # compliance_period = Column(Integer )
+    transaction_effective_date = Column(DateTime, comment='Transaction effective date')
+    transaction_id = Column(Integer, ForeignKey('transaction.id'))
```

Comment:
these foreign key references will need to be updated, for example transaction.transaction_id